py/picca/delta_extraction/masks/lines_mask.py

```diff
@@ -5,9 +5,13 @@
 from picca.delta_extraction.astronomical_objects.forest import Forest
 from picca.delta_extraction.errors import MaskError
 from picca.delta_extraction.mask import Mask
+from picca.delta_extraction.mask import ( # pylint: disable=unused-import
+    accepted_options, defaults)
+from picca.delta_extraction.utils import update_accepted_options
 
-accepted_options = ["filename", "keep pixels"]
-
+accepted_options = update_accepted_options(accepted_options, [
+    "filename"
+])
```

Comment:
keep_pixels is always added in the mask.py now, right?

Reply:
yes, it was read from there before too. We kind of missed it when all these changes were introduced, and it makes more sense that this option is defined here, where it is actually used, as opposed to it being defined in every child class